packages/formstr-sdk/src/form-schemas/v1/form-spec.json

```diff
@@ -87,7 +87,6 @@
   },
   "required": [
     "schemaVersion",
-    "name",
-    "fields"
```

Comment:
why is fields not required?
I cannot see any use case of a form without fields.

Reply:
empty placeholders? templates will be editable